src/hsl_ma57.jl

```diff
@@ -10,6 +10,10 @@ export Ma57Exception
 
 const Ma57Data = Union{Float32, Float64}
 
+const ma57alg = getalg(:hsl_ma57)
+const ma57types = getdatatypes(ma57alg)
+const libma57 = getlib(ma57alg)
```

Comment:
could we keep the name libhsl_ma57 here, so as to minimize changes? Same for ma97.

Reply:
Fixed. If `libhsl_97` is defined it gets the value of `libcoinhsl`.